src/components/atoms/MenuLink/styles.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components"
+import { media } from "../../../styles/breakpoints"
+import { colors, space, fsize } from "../../../styles/vars"
+
+export const StyleNavMenuLink = styled.div`
+    align-items: center;
+    display: flex;
+    min-height: 100px;
```

Comment:
do we need this `min-height`? If this is a button I can see why we need it.

Reply:
Looks like it's ok without min-height. Not sure why I used it, actually.